docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -1,13 +1,9 @@
 # Showcase
 
-<p class="description">The following is a list of some of the public apps using MUI.</p>
+<p class="description">Check out these public apps using Material UI to get inspired for your next project.</p>
 
-Are you using MUI? [Share your product](https://github.com/mui/material-ui/issues/22426).
+Are you working with MUI's component libraries? [Show us what you're building](https://github.com/mui/material-ui/issues/22426)! We'd love to see it.
 
-The websites we include rank well in at least one of these dimensions:
-
-- Non Material Design customization
-- High traffic
-- Open source
+This is a curated list of some of the best apps we've seen that show off what's possible with Material UI.
```

Comment:
Maybe to keep it generic? 

```suggestion
This is a curated list of some of the best apps we've seen that show off what's possible with Material UI or MUI.
```

Reply:
But what does MUI means in that sense? Is it using Base? System? If yes, I guess we should have a Showcase page in each of the packages docs.

Reply:
"MUI" would mean any of our products. We could make it clearer with
```suggestion
This is a curated list of some of the best apps we've seen that show off what's possible with Material UI or other  MUI's products. 
```

Regarding how to scale the showcase, I will expect many projects to use multiple of our products, we should design for this.

Reply:
Yeah, that's a good point, which makes me think that this page might be better suited for an MUI-general documentation space where we could then add a product filter.

Reply:
If we want to keep it open to all of our products, then I would suggest keeping it simple:

```suggestion
This is a curated list of some of the best apps we've seen that show off what's possible with MUI's products.
```